infra/base-images/base-runner/Dockerfile

```diff
@@ -52,6 +53,13 @@ ENV UBSAN_OPTIONS="print_stacktrace=1:print_summary=1:silence_unsigned_overflow=
 ENV FUZZER_ARGS="-rss_limit_mb=2560 -timeout=25"
 ENV AFL_FUZZER_ARGS="-m none"
 
+# Install Rust and cargo-fuzz for libFuzzer instrumentation.
```

Comment:
This should be all there, so now just need rustfilt install add.

Reply:
I see it in base-builder but not base-runner...

Running
`docker run --rm --privileged -i -e FUZZING_ENGINE=libfuzzer -e SANITIZER=address -e RUN_FUZZER_MODE=interactive -v /path/to/whatever:/out -t gcr.io/oss-fuzz-base/base-runner /bin/bash`
and then I get
```
cargo
bash: cargo: command not found
```

Reply:
ok fine to add then. i don't know the reason for this cargo not found.